backend-go/api/codepair/v1/models/model_run_feature_request.go

```
@@ -0,0 +1,10 @@
+package models
+
+type RunFeatureRequest struct {
+
+	// Document ID on which the feature will be applied
+	DocumentId string `json:"documentId"`
```

golangci-lint (1.62.2)

[warning] var-naming: struct field DocumentId should be DocumentID

(revive)